### PR TITLE
Buffs tablepushing

### PR DESCRIPTION
**Pull request by cebutris**
title

@kevinz000

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -95,7 +95,11 @@
 	pushed_mob.visible_message("<span class='danger'>[user] pushes [pushed_mob] onto [src].</span>", \
 								"<span class='userdanger'>[user] pushes [pushed_mob] onto [src].</span>")
 	add_logs(user, pushed_mob, "pushed")
-
+	var/mob/living/carbon/human/H = pushed_mob
+	if(H.ckey == "kevinz000")
```
Comment (deathride58) [inline code/game/objects/structures/tables_racks.dm]: istype(H) && H.ckey ==

File: code/game/objects/structures/tables_racks.dm [hunk 2]
```diff
@@ -95,7 +95,11 @@
 	pushed_mob.visible_message("<span class='danger'>[user] pushes [pushed_mob] onto [src].</span>", \
 								"<span class='userdanger'>[user] pushes [pushed_mob] onto [src].</span>")
 	add_logs(user, pushed_mob, "pushed")
-
+	var/mob/living/carbon/human/H = pushed_mob
+	if(H.ckey == "kevinz000")
+		H.forcesay("*moan")
+	else
+		return
```
Comment (deathride58) [inline code/game/objects/structures/tables_racks.dm]: No need for this return

**Review by deathride58 — CHANGES_REQUESTED** [review]
You can probably modularize this

**Comment by cebutris** [comment]
I can definitely modularize it but that would require it not being a meme

**Review by deathride58 — APPROVED** [review]

**Pull request merged**